src/main/java/com/lpvs/service/scan/LPVSDetectService.java

```diff
@@ -105,52 +111,118 @@ public LPVSDetectService(
-                        String report =
-                                LPVSCommentUtil.buildHTMLComment(
-                                        webhookConfig, scanResult, detectedConflicts);
-                        LPVSCommentUtil.saveHTMLToFile(report, buildReportPath.toString());
-                    } else {
-                        log.error(
-                                "Error: The parent directory '"
-                                        + parentDirectory
-                                        + "' does not exist.");
-                    }
+                detectedConflicts = licenseService.findConflicts(webhookConfig, scanResult);
+                generateReport = true;
+                log.info("Single scan of pull request completed.");
+            } catch (Exception ex) {
+                log.error("Single scan of pull request finished with errors.");
+                log.error("Can't trigger single scan: " + ex.getMessage());
+                SpringApplication.exit(ctx, () -> 0);
+            }
+
+            // Scan option - single scan of local file or folder
+        } else if (localPath != null && !HtmlUtils.htmlEscape(localPath).isEmpty()) {
+            log.info("Triggered single scan of local file(s).");
+            try {
+                licenseService.reloadFromTables();
+                File localFile = new File(localPath);
+                if (localFile.exists()) {
+                    // 1. Generate webhook config
+                    webhookConfig = getInternalQueueByLocalPath();
+                    // 2. Copy files
+                    LPVSFileUtil.copyFiles(
+                            localPath, LPVSFileUtil.getLocalDirectoryPath(webhookConfig));
+                    // 3. Trigger scan
+                    scanResult =
+                            this.runScan(
+                                    webhookConfig,
+                                    LPVSFileUtil.getLocalDirectoryPath(webhookConfig));
+
+                    detectedConflicts = licenseService.findConflicts(webhookConfig, scanResult);
+                    generateReport = true;
+                    log.info("Single scan of local file(s) completed.");
                 } else {
-                    String report =
-                            LPVSCommentUtil.reportCommentBuilder(
-                                    webhookConfig, scanResult, detectedConflicts);
-                    if (report != null && !report.isEmpty()) {
-                        log.info(report);
-                    }
+                    throw new Exception("File path does not exist: " + localPath);
                 }
-                log.info("Single scan completed.");
+
             } catch (Exception ex) {
-                log.error("Single scan finished with errors.");
+                log.error("Single scan of local file(s) finished with errors.");
                 log.error("Can't trigger single scan: " + ex.getMessage());
```

Comment:
What about: 
`log.error("Single scan of local file(s) failed with error: " + ex.getMessage());`